doc/p2p-bad-ports.md

```diff
@@ -0,0 +1,109 @@
+When Bitcoin Core automatically opens outgoing P2P connections it chooses
+a peer (address and port) from its list of potential peers. This list is
+populated with unchecked data, gossiped over the P2P network by other peers.
+
+It may happen that a malicious actor gossips an address:port where no Bitcoin
+node is listening. Or even where another service is listening which is not
+related to the Bitcoin network. As a result, this service may occasionally
+get connection attempts from Bitcoin nodes.
+
+"Bad" ports are ones that are used by services which usually are not opened to
+the public and usually require authentication. A connection attempt (by Bitcoin
+Core, trying to connect because it thinks there is a Bitcoin node on that
+address:port) to such service may be considered a malicious action by an
+ultra-paranoid administrator. An example for such a port is 22 (ssh). On the
+other hand, connection attempts to public services that usually do not require
+authentication are unlikely to be considered a malicious action,
+e.g. port 80 (http).
+
+Following is a list of "bad" ports which Bitcoin Core avoids when choosing a
+peer to connect to. If a node is listening on such a port, it will likely get
+less incoming connections.
```

Comment:
This text could maybe be pithier and has a few grammar issues.

<details><summary>suggestion</summary><p>

```
When Bitcoin Core automatically opens outgoing P2P connections, it chooses a
peer from the potential peers in `peers.dat` gossiped over the P2P network by
other peers.

A malicious actor can gossip an address:port where no Bitcoin node is listening,
or one where a service is listening that is not related to the Bitcoin
network.  These may receive connection attempts from Bitcoin nodes.

"Bad" ports are ones used by services which are usually not open to the public
and require authentication. A connection attempt to such service by a Bitcoin
Core node may be considered a malicious action by an paranoid administrator. An
example for such a port is 22 (ssh). On the other hand, connection attempts to
public services that usually do not require authentication are unlikely to be
considered malicious, e.g. port 80 (http).

Below is a list of "bad" ports which Bitcoin Core avoids when choosing a
peer to connect to. If a node is listening on such a port, it will likely
receive fewer incoming connections.
```
</p></details>

Reply:
Picked some of that, thanks!